core/templates/pages/exploration-editor-page/exploration-editor-page.constants.ts

```diff
@@ -39,7 +39,7 @@ export const ExplorationEditorPageConstants = {
 
   IMPROVE_TYPE_INCOMPLETE: 'incomplete',
 
-  DEFAULT_AUDIO_LANGUAGE: 'en',
+  DEFAULT_TRANSLATION_LANGUAGE: 'ar',
```

Comment:
Note: This change was needed because English is not a valid "translation language" for the default explorations created in tests and in general (which are in English).

Reply:
Actually, wait, I just realized that this would still pose a problem for explorations in Arabic. Not sure how best to handle this -- closing for now.